FindTown/FindTown/Presentation/HomeScene/Home/SubViews/TownTableViewCell.swift

```diff
@@ -152,4 +154,10 @@ final class TownTableViewCell: UITableViewCell {
     @objc func tapIntroduceBtn() {
         introduceBtnAction?()
     }
+    
+    @objc func tapHeartButton() {
+        if !heartIcon.isSelected {
+            heartBtnAction?()
+        }
```

Comment:
```suggestion
        if heartIcon.isSelected == false {
            heartBtnAction?()
        }
```
제 개인적인 스타일인데요! 가독성을 위해 true, false로 코드를 작성하는 것을 선호합니다 ☺️

Reply:
저는 복잡하지 않은건 줄여서 작성하는게 좋더라구요!  저희 프로젝트 내에서 다른 곳에서도 저렇게 작성하고 있어서 이부분은 이대로도 괜찮지 않을까요? ㅎㅎ..

Reply:
네! 각자의 코딩스타일이 있으니까요! 지금도 좋습니다 👍